src/main.cc

```diff
@@ -418,11 +426,7 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
 }
 
 static int cli_install(LiteClient& client, const bpo::variables_map& params) {
-  // Make sure no any other update instances are running, i.e. neither the daemon or the other CLI update/finalize is
-  // running
-  // The API's AkliteClient uses different type of file locking, so it cannot be used for syncing of the daemon and the
-  // CLI command running. Specifically, the API's lock relies on "#define LOCK_EX 2 /* Exclusive lock.  */" while the
-  // daemon's lock is based on "# define F_WRLCK 1 /* Write lock.  */"
+  // Make sure no other update instances are running, i.e. neither the daemon or other CLI update/finalize
   FileLock lock;
```

Comment:
I think this becomes redundant now as the `AkliteClient` ctor at #445 uses the same locking mechanism as the daemon.

Reply:
However, let's leave it, it won't harm and prevent the `LiteClient` instance creation, it may do some "bad" stuff in its ctor.